ci-operator/step-registry/openshift-qe/rosa-loaded-upgrade/openshift-qe-rosa-loaded-upgrade-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
+  as: openshift-qe-rosa-loaded-upgrade
+  steps: 
+  - ref: openshift-qe-rosa-loaded-upgrade
```

Comment:
I think this step and the openshift-qe-rosa-upgrade-postcheck will always want to be used together. Think we can combine the 2 chains to have both ref calls in here

Reply:
+1 
We can add both these steps here and rename chain from `openshift-qe-rosa-loaded-upgrade` to `openshift-qe-rosa-loaded-upgrade-and-postcheck`
    - ref: openshift-qe-rosa-loaded-upgrade
    - ref: openshift-qe-rosa-upgrade-postcheck

Reply:
Update code in new code, thanks!